### DISCUSSION TITLE
Has anyone tried nn-Unet to predict non-binary labels?

### DISCUSSION THREAD

**Discussion by Schobs** [MIC-DKFZ/nnUNet#748] [Q&A] [answered]
I want to input a CxHxW image and out put a 1xHxW map with each pixel a float (0-1). Are there any examples in the package doing something similar that I could look at? Thank you!

**Comment by FabianIsensee** [comment, marked as answer]
unfortunately no - the task you are trying to solve is a regression task and nnU-Net is for segmentation
